plasma_framework/contracts/src/exits/utils/BondSize.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @notice Stores an updateable bond size.
+ */
+library BondSize {
+    uint256 constant public WAITING_PERIOD = 2 days;
+
+    struct Params {
```

Comment:
I think having bond size in wei is less error prone 
i.e. the user can do 
```
bondsize = exitGame.bondSize();
exitGame.startStandardExit(..., { value: bondsize }),
```

without having to remember to convert to gwei

I was going to use a smaller size for the timestamp, but the time constants (e.g. `now`) are uint256 so we'd end up spending a bit more gas on converting them to uint64.
Dunno which is better tbh 🤷‍♂

Reply:
nah, just keep it simple. unless we are packing the three into one slot otherwise we need two slots anyway. I think the optimization now is not as critical as it was on the previous averaging version.